demo/index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Cliquetis demo</title>
+  <style>
+  .log {
+    position: fixed;
+    top: 0;
+    right: 0;
+    width: 50%;
+    height: 100%;
+    padding: .4em;
+    font-size: .8em;
+    color: white;
+  }
+  </style>
+</head>
+<body>
+  <h1>Todo list</h1>
+  <p>Last modified: <span id="lastModified"></span></p>
+  <ul id="todo"></ul>
+  <form>
+    <p>
+      <input type="text" name="label" placeholder="Thing">
+      <input type="submit" value="Add">
+    </p>
+  </form>
+  <p>
+    <input type="button" id="clearCompletedBtn" value="Clear completed">
+    <input type="reset" id="clearAllBtn" value="Clear collection">
+    <input type="button" id="syncBtn" value="Synchronize collection">
+  </p>
+  <p>
+    <textarea id="log" class="log"></textarea>
+  </p>
+  <script src="../dist/cliquetis.js"></script>
+  <script>
+  function main() {
+    var db = new Cliquetis({remote: "http://0.0.0.0:8888/v0"});
+    var tasks = db.collection("todo");
+    var headers = {Authorization: "Basic " + btoa("niko:niko")};
+
+    function toggleDone(task) {
+      return function(event) {
+        task.done = !task.done;
+        tasks.update(task).then(renderList);
+      };
+    }
+
+    function deleteTask(task) {
+      return function(event) {
+        tasks.delete(task.id).then(renderList);
+      }
+    }
+
+    function logResult(status, result) {
+      var log = document.querySelector("#log");
+      if (status !== "OK") {
+        log.style.background = "red";
+      } else {
+        log.style.background = "green";
+      }
+      log.value = status + "\n\n" + JSON.stringify(result, null, 2);
+    }
+
+    function renderList() {
+      tasks.list({}, {includeDeleted: true}).then(function(res) {
+        var ul = document.querySelector("ul");
+        ul.innerHTML = "";
+        res.data.forEach(function(task) {
+          var li = document.createElement("li");
+          li.innerHTML = [
+            '<label>',
+            '  <input type="checkbox" ' + (task.done ? 'checked' : '') + '> ',
+            '  ' + (task.done ? '<del>' + task.label + '</del>' : task.label),
+            '</label>',
+            '<br><small>id: ' + task.id +
+              ', last_modified: ' + task.last_modified +
+              ', _status=' + task._status + '</small>',
+            '<input type="button" value="Delete">'
+          ].join("");
+          li.querySelector("input[type=checkbox]")
+            .addEventListener("click", toggleDone(task));
+          li.querySelector("input[type=button]")
+            .addEventListener("click", deleteTask(task));
+          ul.appendChild(li);
+        });
+      });
```

Comment:
GOOD CATCH, HAHAHAHAHAHA

/me needs some rest, though not HTTP one HAHAHAHAHA (sry)